benchmark/docker-compare.sh

```
@@ -0,0 +1,63 @@
+#
+# The server writes to a 1g tmpfs (RAM disk, so fsync cost is identical across runs). A fresh server is
+# brought up per case so one case's data cannot fill the tmpfs and make the next one fail on a full disk
+# (that artifact previously showed up as phantom dropped connections). For the same reason the measured
+# window is kept short (default warmup 1 + duration 4): at ~500k rec/s a single case writes well under 1g.
+# The per-case row now also prints err/drop/over/recon, so a run that drops connections or sheds load can
+# never be mistaken for a clean result again; on a fresh server all four should read 0.
+#
+# Usage: benchmark/docker-compare.sh   (override the window with DUR=.. WARM=.. if you raise the tmpfs size)
+set -uo pipefail
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+cd "$ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)